wa/workloads/geekbench/uiauto/app/src/main/java/com/arm/wa/uiauto/geekbench/UiAutomation.java

```diff
@@ -184,4 +190,10 @@ public void shareResults() throws Exception {
         shareButton.waitForExists(WAIT_TIMEOUT_5SEC);
         shareButton.click();
     }
+
+    public void scrollPage() throws Exception {
+        UiScrollable listView = new UiScrollable(new UiSelector().className("android.widget.ScrollView"));
+        listView.setMaxSearchSwipes(3);
+        listView.scrollTextIntoView("RUN CPU BENCHMARK");
```

Comment:
Will the list view also be present in geekbench corporate 4? Or should this check for the existence of the scroll view first?

Reply:
list view is present in both.